templates/app/default/template/Resources/app.js

```diff
@@ -1,47 +1,43 @@
 /**
- * Create a new `Ti.UI.TabGroup`.
+ * Create a new tab group.
+ * @see https://titaniumsdk.com/api/titanium/ui/tabgroup.html
  */
-var tabGroup = Ti.UI.createTabGroup();
+const tabGroup = Ti.UI.createTabGroup();
 
 /**
  * Add the two created tabs to the tabGroup object.
  */
-tabGroup.addTab(createTab("Tab 1", "I am Window 1", "assets/images/tab1.png"));
-tabGroup.addTab(createTab("Tab 2", "I am Window 2", "assets/images/tab2.png"));
+tabGroup.addTab(createTab('Tab 1', 'I am Window 1', 'assets/images/tab1.png'));
+tabGroup.addTab(createTab('Tab 2', 'I am Window 2', 'assets/images/tab2.png'));
 
 /**
- * Open the tabGroup
+ * Open the tab group
  */
 tabGroup.open();
 
 /**
- * Creates a new Tab and configures it.
+ * Creates a new tab and configures it
  *
  * @param  {String} title The title used in the `Ti.UI.Tab` and it's included `Ti.UI.Window`
  * @param  {String} message The title displayed in the `Ti.UI.Label`
  * @return {String} icon The icon used in the `Ti.UI.Tab`
  */
 function createTab(title, message, icon) {
-    var win = Ti.UI.createWindow({
-        title: title,
-        backgroundColor: '#fff'
+    const window = Ti.UI.createWindow({
+        backgroundColor: 'backgroundColor', // This is a semantic color defined in the "semantic.colors.json"
+        title: title
     });
 
-    var label = Ti.UI.createLabel({
+    const label = Ti.UI.createLabel({
         text: message,
-        color: "#333",
-        font: {
-            fontSize: 20
-        }
+        color: 'textColor' // This is a semantic color defined in the "semantic.colors.json"
     });
 
-    win.add(label);
+    window.add(label);
 
-    var tab = Ti.UI.createTab({
-        title: title,
-        icon: icon,
-        window: win
-    });
-
-    return tab;
+    return Ti.UI.createTab({
+        title,
+        icon,
+        window
+    });;
```

Comment:
remove one `;`

I would prefer adding the parameter names like `title: title, icon: icon, window: window` since it is a bit easier to understand/quicker to see what it is doing. Both work fine

Reply:
In some linting rules, this is detected as bad style IIRC, so I added it this way. Not sure, though. @janvennemann what do you think?

Reply:
@m1ga I restored the changes as you suggested to prevent the PR from being halted.